documentation/modules/configuring-mtv-operator.adoc

```diff
@@ -66,6 +69,23 @@ spec:
 |`controller_block_overhead`
 |Fixed amount of additional space allocated in persistent block volumes. This setting is applicable for any `storageclass` that is block-based. It can be used when data, such as encryption headers, is written to the persistent volumes in addition to the content of the virtual disk. Note, this setting can only be changed using the CLI.
 |`0`
+
+
+|`vsphere_osmap_configmap_name`
+|Configuration map for vSphere source providers. This configuration map maps the operating system of the incoming VM to a {virt} preference name. This configuration map needs to be in the namespace where `forklift` is deployed, that is, where `forklift-controller` is running. Note, this setting can only be changed using the CLI.
+
+To see the list of preferences in your {virt} environment, open the {ocp-name} web console and click *Virtualization* -> *Preferences*.
+
+An authorized user can overwrite, add, or delete values from the specified configuration map.
```

Comment:
actually, on a second look - this needs further changes (also below for RHV)
we should say something like this: an authorized user can add values to the configuration map when this configuration has the default value, `forklift-vsphere-osmap`. in order to override or delete values, a configuration map that is different from `forklift-vsphere-osmap` should be specified.